conductor_api/src/config.rs

```diff
@@ -304,6 +310,9 @@ pub struct AgentConfiguration {
     pub name: String,
     pub public_address: String,
     pub key_file: String,
+    /// If set to true conductor will ignore key_file and instead use the remote signer
+    /// accessible through signing_service_uri to request signatures.
+    pub holo_remote_key: Option<bool>,
```

Comment:
This is a perfect job for an enum!

```rust

    // ...
    pub key: KeySource
    // ...

enum KeySource = { 
    KeyFile(PathBuf), 
    RemoteKey(String), // then we can put the signing URL directly into the enum 
}
```

Reply:
That is, only if we're going to keep the remote key functionality after the holo hack goes away.